showcase/src/assets/translations/nl.json

```diff
@@ -43,9 +43,11 @@
       },
       "DATE_PICKER": {
         "CUSTOM_FILTER": "Date Picker - Aangepaste filter",
+        "NG_MODEL": "Date Picker - NgModel gebruiken",
         "ONLY_WEEKDAYS": "Date Picker - Alleen weekdagen",
         "ONLY_WEEKENDS": "Date Picker - Alleen weekenden",
         "PLACEHOLDER": "Datum kiezen",
+        "REACTIVE_FORM": "Date Picker - Reactive form gebruiken",
```

Comment:
A literal translation would be "gebruik makend van Reactive form" but I think "Met Reactive form" fits better here